grails-app/services/com/unifina/service/StreamService.groovy

```diff
@@ -202,17 +202,17 @@ class StreamService {
 
 	@CompileStatic
 	private boolean isDirectPermissionToStream(SecUser user, Stream stream) {
-		return permissionService.canRead(user, stream)
+		return permissionService.canReadStream(user, stream)
 	}
 
 	@CompileStatic
 	private boolean isDirectPermissionToStream(Key key, Stream stream) {
-		return permissionService.canRead(key, stream)
+		return permissionService.canReadStream(key, stream)
 	}
 
 	@CompileStatic
 	private boolean isPermissionToStreamsUiChannelCanvas(SecUser user, Stream stream) {
-		return stream.uiChannel && stream.uiChannelCanvas && permissionService.canRead(user, stream.uiChannelCanvas)
+		return stream.uiChannel && stream.uiChannelCanvas && permissionService.canReadCanvas(user, stream.uiChannelCanvas)
```

Comment:
The method name is vague, but I think the intent here is to `check(user, stream.uiChannelCanvas, CANVAS_GET)`